tests/test_pay.py

```diff
@@ -150,7 +150,7 @@ def test_pay_get_error_with_update(node_factory):
     # channel_update, and it should patch it to include a type prefix. The
     # prefix 0x0102 should be in the channel_update, but not in the
     # onionreply (negation of 0x0102 in the RE)
-    l1.daemon.wait_for_log(r'Extracted channel_update 0102.*from onionreply 10070080(?!.*0102)')
+    l1.daemon.wait_for_log(r'Extracted channel_update 0102.*from onionreply 10070088[0-9a-fA-F]{88}')
```

Comment:
By setting the `message_flags` to 1 for the optional htlc_maximum_msat, the flags portion of the `channel_update` causes this to fail. Additionally, the message is now 88 bytes long.  Instead, we just check that the onionreply has the correct prefix and actual length.

Reply:
For future reference: I think this note belongs in the commit message; I consider github transitory.  Plus, it's a fun story, and such tales are the intermittent reward for reading the git log :)

Reply:
:+1: